sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -148,15 +155,17 @@ export interface EventHubClientOptions {
   webSocketConstructorOptions?: any;
   /**
    * @property
-   * Retry options for all the operations on the client/producer/consumer.
+   * The retry options for all the operations on the client/producer/consumer.
    * This can be overridden by the retry options set on the producer and consumer.
    */
   retryOptions?: RetryOptions;
 }
 
 /**
- * @class EventHubClient
- * Describes the EventHub client.
+ * @class
+ * The client is the main point of interaction with Azure Event Hubs service.
+ * It offers connection to a specific Event Hub within the Event Hubs namespace and
+ * offers operations for sending event data, receiving events, and inspecting the connected Event Hub.
```

Comment:
Is the second "offers" in the same sentence needed?

Reply:
Removed!